thumbnails/CHANGELOG.md

```diff
@@ -0,0 +1,147 @@
+# Changelog for [0.3.0] (2020-08-17)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)